rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+while maintaining privacy and enabling proofs of correctness in transactions. The commitment formula is as follows:
+
+$$Commitment = C(UTXO,RCK) =g^{UTXO}⋅h^{RCK}$$
+
+
+In this formula, $g$ and $h$ are two generators of a cryptographic group where no known relationship exists between them. This ensures that the commitment is secure 
+and computationally infeasible to reverse or manipulate without knowing the original UTXO components. The random number $RCK$ adds an additional layer of security 
+by blinding the UTXO's contents, ensuring that the commitment doesn't leak any information about the underlying data.
+
+**Importance of Homomorphic Commitments**
+It is essential to use a homomorphic commitment like the Pedersen commitment for UTXOs because it allows for the verification of important properties in transactions, 
+such as ensuring that the total input value of a transaction equals the total output value. This balance is crucial for preventing unauthorized creation of funds or d
+iscrepancies in transactions. A homomorphic commitment enables these proofs because of its additive properties. Specifically, the exponents in the commitment formula are additive, 
+meaning that commitments can be combined and verified without revealing the individual components. For instance, if you have two UTXOs with commitments $C(UTXO_1,RCK_1)$ 
+and $C(UTXO_2,RCK_2)$, you can combine them and verify that the resulting commitment is valid without exposing the actual amounts.
+
+This capability is leveraged through a modified version of the Schnorr protocol, which is used in conjunction with the Pedersen hash to verify the correctness of transactions. 
+The Schnorr protocol allows users to prove, without revealing the actual values, that the sum of inputs equals the sum of outputs, ensuring that no funds are created or lost in the transaction.
+
+**Limitations of Standard Cryptographic Hashes**
+Standard cryptographic hash functions, such as SHA-256, are not suitable for this purpose because they lack the algebraic structure needed for homomorphic properties. 
+In particular, while SHA-256 provides strong security for general hashing purposes, it does not allow the additive properties that are required to perform the type of 
+zero-knowledge proofs used in Nescience for UTXO commitments. This is why the Pedersen hash is preferred, as it enables the secure and private execution of transactions
+while allowing for balance verification and other critical proofs.
+
+**Conclusion**
+By using homomorphic commitments like the Pedersen hash, the Nescience architecture ensures that UTXOs can be securely committed and validated without exposing sensitive information. 
+The random component (RCK) adds an additional layer of security, and the additive properties of the Pedersen commitment enable powerful zero-knowledge proofs that maintain the 
```

Comment:
I suggest removing the parentheses around RCK and changing to:
"The random component, RCK, adds" - this is to avoid confusion over what our self-defined RCK acronym